client/lib/signup/step-actions.js

```diff
@@ -157,6 +157,13 @@ export function createSiteWithCart(
 		newSiteParams.blog_name = importingFromUrl;
 		newSiteParams.find_available_url = true;
 		newSiteParams.public = -1;
+	} else if (
+		flowName === 'onboarding' &&
+		'remove' === getABTestVariation( 'removeDomainsStepFromOnboarding' )
```

Comment:
Should `config.isEnabled( 'signup/domains-step-removal' ) &&` be here too?

Reply:
It doesn't need here because no user is assigned to `removeDomainsStepFromOnboarding` because of the feature flag.

Reply:
because of ~the feature flag.~ the 100% control variation.